src/languages/pyret/ast.js

```diff
@@ -101,15 +106,19 @@ export class Func extends ASTNode {
   }
 
   pretty() {
-    return P.lambdaLikeSexpr(this.id, this.args, this.body);
+    return P.horzArray([P.txt("fun "), this.name, P.txt("("), P.horzArray(this.args.map(p => p.pretty())), P.txt(")"), this.body]);
```

Comment:
Should (1) have print in two different ways, depending on whether it fits on one line, and (2) make a helper function to display a comma-separated list for the args.

Reply:
updated in new file, so resolving